release 1.0.0 (#608) (#609)
redhat-developer/intellij-kubernetes: README.md

 - Edit Kubernetes resource manifests and apply them to your cluster
 - View Kubernetes and OpenShift clusters in an explorer tree view
 - Tree View supports dynamic update of K8s resources
 - Push and Pull Feature to keep the editor in sync with the Kubernetes resource

>  - Push multiple resources in an editor to create several resources at the time

Reviewer: ```suggestion
 - Allows users to Push multiple resources in an editor to create several resources at once.
```

![editor toolbar push](images/editor-toolbar-push.png)

Alternatively the editor notifies you whenever you may push your content to the cluster.

> Clicking the `Push` link in the editor notification saves your editor to the cluster.

Reviewer: ```suggestion
Clicking the `Push` link in the editor notification saves your file changes on to the cluster.
```

#### Pull (Load) from Cluster

> You can `Pull` the resource from the cluster into you editor. 

Reviewer: ```suggestion
You can `Pull` the resource from the cluster into the local file opened in the editor. 
```

> You click the `Pull` icon in the toolbar and replace the content of your editor with the version that exists on the cluster.

Reviewer: ```suggestion
Clicking on the `Pull` icon in the toolbar replaces the content of your local file in the editor with the version that exists on the cluster.
```
